scRNA-seq/00-scRNA_introduction.md

```diff
@@ -1,39 +1,29 @@
 
-For the purposes of this tutorial, we'll group single-cell technologies into two categories based on their capture methods and quantitative nature.
-The pre-processing steps and the biases to look out for in post-processing vary based on technology and how the cells are sorted.
+Single-cell technologies can be divided into two categories, tag-based and full-length, based on their capture methods and quantitative nature.
 
-For more extensive background on single-cell experimental methods, Kiselev _et al._ have a very good [tutorial for scRNA-seq](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html).
+In *Tag-based* scRNA-seq, cells are separated by emulsion/droplets, and individual cells are given a unique cell barcode prior to sequencing.
+An example of tag-based scRNA-seq is 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601). 
 
-![**Overall view of full-length and tag-based workflows**](diagrams/overview_workflow.png)
+In *full-length* scRNA-seq, cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
+With full-length scRNA-seq, each cell is sequenced individually and has its own fastq file. 
+An example of full-length scRNA-seq is Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006).
 
-### Full-length scRNA-seq  
-*Example:* Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006)    
-Cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
-Each cell is then sequenced individually and has its own fastq file.
-(This will result in two fastq files if this is paired-end sequencing.)
-The data preprocessing steps for these types of scRNA-seq data are similar to bulk RNA-seq methods.
+For the purposes of this tutorial, we will focus on tag-based scRNA-seq, but it is important to keep in mind that the pre-processing steps and the biases to look out for in post-processing vary based on technology and how the cells are sorted.
 
-#### Pros:  
-- Can be paired-end sequencing which has less risk for 3' bias.  
-- More complete coverage of transcripts, which may be better for transcript discovery purposes.   
+For more extensive background on single-cell experimental methods, Kiselev _et al._ have a very good [tutorial for scRNA-seq](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html).
 
-#### Cons:  
-- Is not very efficient (generally 96 cells per plate).  
-- Takes much longer to run (days/weeks depending on sample size).
-- Much more expensive.  
+![**Overall view of scRNA-seq tag-based workflow**](diagrams/overview_workflow.png)
 
 ### Tag-based scRNA-seq  
 *Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)    
-Cells are separated by emulsion/droplets, and individual cells are given barcodes.
```

Comment:
This point seems important to keep! I think it might be worth laying out with even a bit more detail than we had, but the separation step should be there, and the fact that there are (at least) two barcodes